### PR TITLE
Selenium WD Advanced with JS: actions, JS executor

### PR DESCRIPTION
**Pull request by alex39292**
Task - [UI Testing] - Selenium WD Advanced with JS: actions, JS executor

### CODE REVIEW

**Comment by alex39292** [comment]
I think Task "Basic knowledge of TA Frameworks" is already done in this branch.

File: test/module_6/page/app.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-undef */
+const { browser, element, by, $$ } = require("protractor");
```
Comment (SiarheiRylach) [inline test/module_6/page/app.js]: not necessary

File: .mocharc.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const { reporters } = require("mocha");
```
Comment (SiarheiRylach) [inline .mocharc.js]: not necessary

File: configurations/conf.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+exports.config = {
+    framework: "jasmine",
+    specs: ["../test/module_6/protractorTask.spec.js"],
+    capabilities: {
+        browserName: "firefox"
+    }
```
Comment (SiarheiRylach) [inline configurations/conf.js]: Please set screen size inside onPrepare field of config

File: test/module_6/page/app.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        this.text = $("div:nth-child(1) > h3:nth-child(1)");
+        
+    }
+
+    get() {
+        browser.get("http://juliemr.github.io/protractor-demo/");
+    }
+
+    getTitle() {
+        browser.executeScript("document.querySelector('body').style.background = 'grey'");
```
Comment (SiarheiRylach) [inline test/module_6/page/app.js:17]: for future: I would recommend using async/await or promise chaining for async functions. I wouldn't hope on control flow.

File: test/module_6/page/app.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        this.secondNumber.sendKeys(num2);
+        this.operator.sendKeys(operator);
+        browser.actions().mouseMove(this.goButton).click().perform();
+    }
+
+    getResult() {
+        return element(by.binding("latest")).getText();
+    }
+
+    checkHistory() {
```
Comment (SiarheiRylach) [inline test/module_6/page/app.js]: getHistory

Comment (alex39292) [reply test/module_6/page/app.js]: fixed

File: test/module_6/page/app.js (hunk 1 above)
Comment (alex39292) [reply test/module_6/page/app.js]: deleted

File: .mocharc.js (hunk 2 above)
Comment (alex39292) [reply .mocharc.js]: deleted

File: configurations/conf.js (hunk 3 above)
Comment (alex39292) [reply configurations/conf.js]: Set screen size

File: test/module_6/page/app.js (hunk 4 above)
Comment (alex39292) [reply test/module_6/page/app.js:17]: ok)

**Comment by alex39292** [comment]
Fixed

**Comment by SiarheiRylach** [comment]
> I think Task "Basic knowledge of TA Frameworks" is already done in this branch.

Agree. But also I would recommend checking how your comrades performed this task and how they use BaseElement and BasePage.

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**